posthog/api/test/test_organization_feature_flag.py

```diff
@@ -44,3 +51,97 @@ def test_get_feature_flag_unauthorized(self):
         response = self.client.get(url)
 
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+
+class TestOrganizationFeatureFlagCopy(APIBaseTest, QueryMatchingTest):
+    def setUp(self):
+        self.team_1 = self.team
+        self.team_2 = Team.objects.create(organization=self.organization)
+
+        self.feature_flag_key = "copied-flag-key"
+        self.feature_flag_to_copy = FeatureFlag.objects.create(
+            team=self.team_1,
+            created_by=self.user,
+            key=self.feature_flag_key,
+            filters={"groups": [{"rollout_percentage": 65}]},
+        )
+
+        super().setUp()
+
+    @snapshot_postgres_queries
+    def test_copy_feature_flag_success(self):
+        url = f"/api/organizations/{self.organization.id}/feature_flags/copy_flags"
+        data = {
+            "feature_flag_key": self.feature_flag_to_copy.key,
+            "from_project": self.feature_flag_to_copy.team_id,
+            "target_project_ids": [self.team_2.id],
+        }
+        response = self.client.post(url, data)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertIn("success", response.json())
+        self.assertIn("failed", response.json())
+
+        # copied flag is present in the response
+        self.assertEqual(self.feature_flag_key, response.json()["success"][0]["key"])
+
+        expected_fields = {
+            "key": self.feature_flag_to_copy.key,
+            "name": self.feature_flag_to_copy.name,
+            "filters": self.feature_flag_to_copy.filters,
+            "active": self.feature_flag_to_copy.active,
+            "rollout_percentage": self.feature_flag_to_copy.rollout_percentage,
+            "ensure_experience_continuity": self.feature_flag_to_copy.ensure_experience_continuity,
+        }
+        copied_flag = FeatureFlag.objects.get(team=self.team_2, key=self.feature_flag_key)
+        for field, expected_value in expected_fields.items():
+            actual_value = getattr(copied_flag, field)
+            self.assertEqual(expected_value, actual_value)
```

Comment:
this is good!

For the others, I meant checking they are null. Like `analytics_dashboard`, it should be null, and `has_enriched_analytics` should be null too. (same for all others)

Also, add another test which checks the update path (this is just the create path if I understand correctly) - i.e. flag already exists, all values are different, `analytics_dashboard` exists, and then on the copy, `analytics_dashboard` is not null, and is same as previous dashboard.